README_hi.md

```diff
@@ -0,0 +1,248 @@
+<h1 align="center">कंप्यूटर विज्ञान</h1>
+
+## विषय तालिका
+
+- [प्रस्तावना](#introduction)
```

Comment:
change table of contents links:
```
[प्रस्तावना](#introduction) to [प्रस्तावना](#प्रस्तावना)
```

Do this for all.

Reply:
Done

Reply:
I have not completed the translation of the whole document, but I will try to complete it ASAP

Reply:
Done